libevmasm/Assembly.h

```diff
@@ -86,6 +86,9 @@ class Assembly
 	/// Returns the assembly items.
 	AssemblyItems const& items() const { return m_items; }
 
+	/// Returns the mutable assembly items.
+	AssemblyItems& itemsMutable() { return m_items; }
```

Comment:
Any alternative notation `Assembly.h` uses is to overload: `AssemblyItems& items() {..}`.

Reply:
I actually think that a const_cast in a test is better than providing a mutable getter in the general interface.

Reply:
That was the last option I've considered. I can change it, but note, the Assembly interface already exports immutable and mutable versions of the same in another case:
```c++
        Assembly const& sub(size_t _sub) const { return *m_subs.at(_sub); }
        Assembly& sub(size_t _sub) { return *m_subs.at(_sub); }
```

Reply:
Yes, of the assembly interface, but not of the underlying assembly item array, or does it?